tsdb/engine/tsm1/tombstone.go

```diff
@@ -78,6 +79,8 @@ func (t *Tombstoner) TombstoneFiles() []FileStat {
 	return nil
 }
 
+var is_windows = "windows" == runtime.GOOS
```

Comment:
This should probably be done by creating specific operating system files and defining each method in that specific file.